docs/rules/one-var.md

```diff
@@ -45,6 +45,7 @@ Object option:
 * `"let": "never"` requires multiple `let` declarations per block
 * `"const": "always"` requires one `const` declaration per block
 * `"const": "never"` requires multiple `const` declarations per block
+* `"separateRequires": true` enforces `requires` to be separate from declarations
```

Comment:
what about `import`?

Reply:
at this point, it only check function `require()`, that's say, don't have to be `env: {node: true}`, and didn't check `imports`.

Reply:
Right, I’m wondering why we’d want to add the option but without those features.

Reply:
I'm confused about the question -- it seems like `one-var` wouldn't be applicable to imports because imports are not variable declarations.

Reply:
@not-an-aardvark ha, totally fair point there, disregard that :-) what about the node env? Does it make sense to check for requires when the env isn't node?

Reply:
I have no strong feelings either way about that. I think we're mainly adding this for JSCS compatibility, so ideally I think we should match whatever JSCS did in this case.

Reply:
I think that check would be incorrect. People use `require` with browserify/webpack to write browser code. We also do not have that check in other rules that deal with `require`.